src/Silk.Core/Commands/Server/TagCommand.cs

```diff
@@ -220,6 +221,22 @@ public async Task Search(CommandContext ctx, string tagName)
             }
             
         }
+
+        [Command]
+        public async Task Raw(CommandContext ctx, string tag)
+        {
+            Tag? dbTag = await _tagService.GetTagAsync(tag, ctx.Guild.Id);
+
+            if (dbTag is null)
+            {
+                await ctx.RespondAsync("Tag not found!");
+            }
+            else
+            {
+                await ctx.RespondAsync(Formatter.Sanitize(dbTag.Content));
```

Comment:
This should be Formatter.Escape(), since we want to see what formatting is actually applied.

Reply:
Apparently I'm stupid and don't know how the formatter works.